### PR TITLE
update angular quick start guide for angular 17

### PR DESCRIPTION
**Pull request by ankur-dwivedi**

### CODE REVIEW

File: getting-started/quickstart-guides/angular.md [hunk 1]
```diff
@@ -698,19 +718,14 @@ If you GET `http://localhost:3000/auth`, you should see a JSON response like thi
 
 Now that we have our authentication server up and running, let's configure the `publicKey` and `authenticator` in the frontend Angular app:
 
-Add the following to `src/app/app.module.js` file to [initialize the SDK](angular.md#initialize-the-angular-sdk) with auth params:
-
-```jsx
-publicKey: "<YOUR_PUBLIC_KEY>",
-urlEndpoint: "YOUR_ENDPOINT",
-```
+Before proceeding, ensure you've [initialize the SDK](angular.md#initialize-the-angular-sdk) with authentication parameters. Open the `src/app/app.module.js` file and replace `your_public_key` and `your_endpoint` with your actual `publicKey` and `urlEndpoint` values.
```
Comment (ahnv) [inline getting-started/quickstart-guides/angular.md]: Shouldn't this be app.module.ts ?

File: getting-started/quickstart-guides/angular.md [hunk 2]
```diff
@@ -698,19 +718,14 @@ If you GET `http://localhost:3000/auth`, you should see a JSON response like thi
...
 
 Add the following to `src/app/app.component.js`:
```
Comment (ahnv) [inline getting-started/quickstart-guides/angular.md]: similarly here app.component.ts?

Comment (ankur-dwivedi) [reply getting-started/quickstart-guides/angular.md]: done

File: getting-started/quickstart-guides/angular.md (hunk 1 above)
Comment (ankur-dwivedi) [reply getting-started/quickstart-guides/angular.md]: done

**Review by ahnv — APPROVED** [review]

**Pull request merged**